AGENTS.md

```
@@ -183,120 +183,117 @@ src/
 |---------|-------------|
-| `gssh identity init` | Create machine identity keypair |
-| `gssh identity show` | Display identity fingerprint |
-| `gssh access add <pubkey>` | Add authorized client |
-| `gssh access list` | List authorized clients |
-| `gssh access remove <key>` | Remove client access |
-| `gssh share create` | Create invite token |
+| `gssh user identity init` | Create user root identity |
+| `gssh user identity show` | Display identity fingerprint |
+| `gssh machine access add <gssh-user:...>` | Grant machine full access |
+| `gssh machine access list` | List machine collaborators |
+| `gssh machine access remove <user-id|label>` | Remove machine full access |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)